### PR TITLE
Refactor fix trivial cpp coreguideline violations

### PR DESCRIPTION
**Pull request by Swiftb0y**
see individual commit messages

### CODE REVIEW

File: src/analyzer/analyzergain.cpp [hunk 1]
```diff
@@ -7,18 +7,14 @@
...
 #include "analyzer/constants.h"
 #include "track/track.h"
-#include "util/math.h"
 #include "util/sample.h"
 #include "util/timer.h"
 
 AnalyzerGain::AnalyzerGain(UserSettingsPointer pConfig)
-        : m_rgSettings(pConfig) {
-    m_pReplayGain = new ReplayGain();
+        : m_rgSettings(pConfig), m_pReplayGain(std::make_unique<ReplayGain>()) {
```
Comment (daschuer) [inline src/analyzer/analyzergain.cpp:14]: We usually stack class initialization lists.

**Review by daschuer — APPROVED** [review]
Just a minor code style comment. Do you want to "fix" it?

File: src/analyzer/analyzergain.cpp (hunk 1 above)
Comment (Swiftb0y) [reply src/analyzer/analyzergain.cpp]: like in the newest latest commit?

Comment (daschuer) [reply src/analyzer/analyzergain.cpp]: Yes. Thank you.

**Comment by Swiftb0y** [comment]
Thanks.

**Pull request merged**